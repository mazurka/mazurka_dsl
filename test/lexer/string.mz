"this is a string"

"
multiline

string
"