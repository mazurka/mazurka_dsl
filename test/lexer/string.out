{string,1,<<"this is a string">>}.
{string,3,<<"\nmultiline\n\nstring\n">>}.